README.md

```diff
@@ -1,73 +1,72 @@
-mind that as these are probably not part of the role API they may change during
-the lifetime.
-
-Example of setting the variables:
-
-```yaml
-template_foo: "oof"
-template_bar: "baz"
-```
-
-### Variables Exported by the Role
-
-This section is optional.  Some roles may export variables for playbooks to
-use later.  These are analogous to "return values" in Ansible modules.  For
-example, if a role performs some action that will require a system reboot, but
-the user wants to defer the reboot, the role might set a variable like
-`template_reboot_needed: true` that the playbook can use to reboot at a more
-convenient time.
-
-Example:
-
-`template_reboot_needed` - default `false` - if `true`, this means
-a reboot is needed to apply the changes made by the role
-
-## Dependencies
-
-A list of other roles hosted on Galaxy should go here, plus any details in
-regards to parameters that may need to be set for other roles, or variables
-that are used from other roles.
-
-## Example Playbook
-
-Including an example of how to use your role (for instance, with variables
-passed in as parameters) is always nice for users too:
-
-```yaml
-- hosts: all
-  vars:
-    template_foo: "foo foo!"
-    template_bar: "progress bar"
-
-  roles:
-    - linux-system-roles.template
-```
+### VPN global variables (internal)
+
+System role’s global internal variables, which affect the whole system role. No platform specific variables have been defined yet.
+
+| Parameter               | Description                                         | Type |
+|-------------------------|-----------------------------------------------------|:----:|
+| vpn\_packages           | A list of packages required to set up a VPN tunnel. | list |
+
+### VPN global variables (user-defined)
+
+System role’s global variables, which affect the whole system role and can be defined by the user in the inventory file provided by them.
+
+| Parameter            | Description                                                                                       | Type        | Required | Default                 |
+|----------------------|---------------------------------------------------------------------------------------------------|:-----------:|:--------:|-------------------------|
+| vpn\_provider        | The VPN provider to use (e.g. libreswan, wireguard, etc.)                                         | str         | no       | libreswan               |
+| vpn\_enc\_alg        | The VPN encryption algorithm to use                                                               | str         | no       | -                       |
+| vpn\_lifetime        | How long a VPN instance should last before being negotiated again.                                | int         | no       | -                       |
+| vpn\_connections     | Email (or list of emails) to be included in the certificate.                                      | dict        | yes      | -                       |
```

Comment:
copy/paste from certificate role

Reply:
Can you elaborate a little more on what you mean?

Reply:
Sorry - it looks like this has been copied from the certificate role "Email (or list of emails) to be included in the certificate." - You should change this to be the description for the VPN role parameter

Reply:
Oh sorry, I missed that. Should be resolved now